Library/Formula/luabind.rb

```diff
@@ -58,6 +58,22 @@ def install
     end
     args << "--prefix=#{prefix}"
     system "bjam", *args
+
+    (lib/"pkgconfig/luabind.pc").write pc_file
+  end
+
+  def pc_file; <<-EOS.undent
```

Comment:
Since this is just used once it doesn't really need to be a method; may as well just do 
```ruby 
(lib/"pkgconfig/luabind.pc").write <<-EOS.undent
  prefix=#{HOMEBREW_PREFIX}
```